### PR TITLE
Add env vars for Workspace, Region and Branch

### PR DESCRIPTION
**Pull request by philkra**
The workspace Id can be set via env var `XATA_WORKSPACE_ID`. This env var takes precedence over the `.xatarc` file. The `XATA_WORKSPACE_ID` is present, there is an additional lookup for the env vars `XATA_BRANCH` and `XATA_REGION`.

### CODE REVIEW

File: xata/utils.go [hunk 1]
```diff
@@ -161,11 +161,11 @@ func loadConfig(fieName string) (config, error) {
...
-// getBranchName retrieves the branch name.
-// If not found, falls back to defaultBranchName
-func getBranchName() string {
-	if branchName, found := os.LookupEnv(branchNameEnvVar); found {
-		return branchName
+// Get value from env var with fallback to godotenv
+// return default value if not found
+func getEnvVar(name string, defaultValue string) string {
+	if val, found := os.LookupEnv(name); found {
+		return val
```
Comment (ryanslade) [inline xata/utils.go:168]: I noticed that if the .env file doesn't exist we return an empty string instead of the default value, is that correct?

File: xata/utils.go [hunk 2]
```diff
@@ -177,11 +177,22 @@ func getBranchName() string {
...
+// getBranchName retrieves the branch name.
+// If not found, falls back to defaultBranchName
+func getBranchName() string {
+	return getEnvVar(branchNameEnvVar, defaultBranchName)
+}
+
+// Get the region if the corresponding env var `XATA_REGION` is set
+// otherwise return the default region: us-east-1
+func getRegion() string {
+	return getEnvVar("XATA_REGION", defaultRegion)
```
Comment (ryanslade) [inline xata/utils.go:195]: nit: We appear to sometimes uses the raw string value of the environment variable names and sometimes constants. I think we should switch to constants everywhere. I'm happy to do this as a separate PR as doing it as part of this one could get messy.

**Review by ryanslade — APPROVED** [review]

File: xata/utils.go (hunk 1 above)
Comment (philkra) [reply xata/utils.go:168]: Yes, that is correct. It's fine for now, as we only fetch an env var to bootstrap the connection params and if an empty string is encountered, continues to probe the `.xatarc` file.

File: xata/utils.go (hunk 2 above)
Comment (philkra) [reply xata/utils.go:195]: that's a good nit, I felt guilty not doing it :) I'll push it up in a sec and share the commit.

Comment (philkra) [reply xata/utils.go]: https://github.com/xataio/xata-go/pull/54/commits/52291e0a0da1970c3cbe9260ef44c8409ee26c72

Comment (ryanslade) [reply xata/utils.go]: We still use the string values in a lot of tests. To get around this we'd have to make the constants public.
